specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2024-10-01-preview/definitions.json

```diff
@@ -9232,13 +9232,60 @@
         "description": {
           "type": "string",
           "description": "Description of the workspace."
+        },
+        "kind": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The kind of workspace.",
+          "enum": [
+            "KubernetesManaged",
+            "Default"
+          ],
+          "x-ms-enum": {
+            "name": "GatewayKind",
```

Comment:
@solankisamir Should we rename this to WorkspaceKind given this is on workspace resource, or should we keep calling it GatewayKind to keep enums aligned in generated code?

I'm inclined to say - Should be renamed to `WorkspaceKind`?

Reply:
Changed to `WorkspaceKind` as I agree that it looks better to match the resource. Will revert if advised otherwise.